src/Document/TextPathElement.ts

```diff
@@ -42,8 +41,8 @@ interface IEquidistantCache {
 }
 
 interface IGlyphInfo {
-	transposeX: number;
-	transposeY: number;
+	// transposeX: number;
+	// transposeY: number;
```

Comment:
remove from code if not using?

Reply:
```
// Note: Since glyphs are rendered one at a time, any kerning pair data built into the font will not be used.
// Can foresee having a rough pair table built in that the developer can override as needed.
// Or use "dx" attribute of the <text> node as a naive replacement
// const kern = 0;
// placeholder for future implementation
```

I think this code can be uncommented and used in the future.